specs/payment_version.md

```diff
@@ -0,0 +1,17 @@
+### Added possibility of setting version of payment protocol
+
+A new field was added to offer on provider side and demand on requestor side
+golem.com.payment.protocol.version
+There is only one possible value for now: 2
+
+```golem.com.payment.protocol.version = 2```

```

Comment:
Property should be mentioned in https://github.com/golemfactory/golem-architecture/blob/master/standards/3-commercial/com/payment.md and should link to this page

Reply:
I don't treat this comment as resolved. Changes that were made there don't conform to structure of the document. Each property has separate paragraph with description that has very specific format.
The second thing is that you mentioned protocol version in article about something completely different in the description of choosing platform